source/index.html.md

```diff
@@ -333,38 +333,48 @@ Example response (snippet):
     },
     "processed_at": "2018-07-09T12:58:56+00:00",
     "amount": 50000,
-    "currency": "EUR",
-    "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
+    "currency": "EUR"
 }
 ````
 
 
-## Recurring payments
+## Credential on file
 
-Recurring payments enable you to repeatedly charge cardholders without having
-them to provide card information for subsequent payments.
+Subscriptions are supported using `credential_on_file[]`. There are 2 types of
+subscriptions: `recurring` and `unscheduled` (merchant-initiated) credential on
+file (UCOF).
 
-### Subscription concept
+A `recurring` series consists of transactions with the same amount being charged
+at regular intervals.
 
-Many PSPs have a subscription concept for supporting recurring payments.
-The first approved authorization is the initial recurring authorization (also
-known as "first in series"), all later authorizations are called subsequent
-authorizations. Clearhaus supports subscriptions in the form of recurring
-payments.
+An `unscheduled` credential on file is a series of merchant-initiated
+transactions (MIT) just like a `recurring` series, but with the exception that
+the intervals between withdrawals are irregular; instead, transactions are made
+ad-hoc.
 
-### Repeatedly reserve money
+For subscriptions, the first in series transaction is a cardholder-initiated
+transaction (CIT) whereas all subsequent transactions are merchant-initiated.
 
-A recurring payment is made by making an authorization and setting `recurring`
-parameter to `true`. The first recurring payment for a given card could be made
-this way (notice that the amount may be zero):
+PSPs and merchants may store the cardholder's payment credentials to allow a
+returning cardholder to conveniently pay without entering the payment
+credentials again. (Explicit agreement with Clearhaus is required.)
+To store payment credentials and use these stored payment credentials,
+`credential_on_file[]` must be used. All transactions in such a series are
+cardholder-initiated.
+
+### Example
+
+As an example, a `recurring` payment is made by making an authorization and
+signaling `credential_on_file[type]=recurring`. The first recurring payment for
+a given card is made this way (notice that the amount may be zero):
```

Comment:
Isn't this also the case for `cardholder`? I mean, the CSC could be there, but likely would not be.

Reply:
I guess issuers expect the CSC (at least they seem to be allowed to decline CITs without CSC).

Over at DIBS they seem to require CSC:

> Dine kunder kan vælge at gemme deres kortoplysninger i et DIBS, Nets eller Paytrail betalingsvindue og ved hjælp af kun deres 3-cifrede CVC/CVV-kode genbruge oplysningerne i alle andre DIBS, Nets eller Paytrail-ejede betalingsvinduer.

(https://www.dibs.dk/husk-mig)